stl/src/tzdb.cpp

```diff
@@ -0,0 +1,71 @@
+// Copyright (c) Microsoft Corporation.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+#include <internal_shared.h>
+#include <xtzdb.h>
+
+#include <Windows.h>
+
+#pragma comment(lib, "Advapi32")
+
+_EXTERN_C
+
+_RegistryLeapSecondInfo* __stdcall __std_tzdb_get_reg_leap_seconds(
+    const size_t prev_reg_ls_size, size_t* current_reg_ls_size) {

```

Comment:
My understanding was that it's not required here, because it's compiled into the library, rather than exposed to user code. `xtime.cpp`, for example, follows this pattern. But I'm happy to change it if you've settled on a convention of `_Ugly` everywhere.

Reply:
Ah good point -- I was finding some conflicting conventions in similar files.  You make good points and since there seems to be some of each convention already in existence, I think we can leave this as-is!